src/processor/map/map_copy_from.cpp

```diff
@@ -19,10 +19,8 @@ namespace processor {
 std::unique_ptr<PhysicalOperator> PlanMapper::mapCopyFrom(LogicalOperator* logicalOperator) {
     auto copyFrom = (LogicalCopyFrom*)logicalOperator;
     auto info = copyFrom->getInfo();
-    if (info->copyDesc->fileType == CopyDescription::FileType::TURTLE) {
-        throw NotImplementedException("PlanMapper::mapCopyFrom");
-    }
     switch (copyFrom->getInfo()->tableSchema->getTableType()) {
+    case TableType::RDF:
```

Comment:
This is probably wrong because the table type you get will be node for the following statement
```
COPY taxonomy_RESOURCE FROM "dataset/copy-test/node/turtle/taxonomy.ttl" ;
```
I guess that's also why the rehash part is not being triggered

Reply:
Actually, i don't need add this case statement